lib/Conversion/SCFToCalyx/SCFToCalyx.cpp

```diff
@@ -514,6 +515,13 @@ class BuildOpGroups : public calyx::FuncOpPartialLoweringPattern {
                                /*subtract=*/1);
       }
       rewriter.create<calyx::AssignOp>(loc, opFOp.getSubOp(), subOp);
+    } else if (isa<calyx::DivSqrtOpIEEE754>(opPipe)) {
+      auto opFOp = cast<calyx::DivSqrtOpIEEE754>(opPipe);
```

Comment:
```suggestion
    } else if (auto opFOp = dyn_cast<calyx::DivSqrtOpIEEE754>(opPipe)) {
```

Reply:
this could not be applied as the compiler failed type checking :/